src/transformers/models/t5/modeling_flax_t5.py

```diff
@@ -658,6 +684,7 @@ def __call__(
         encoder_attention_mask=None,
         output_attentions: bool = False,
         output_hidden_states: bool = False,
+        return_dict: bool = True,
```

Comment:
Note that we don't actually need to propagate `return_dict` to the `FlaxT5LayerCollection` - this arg is not used by the `FlaxT5LayerCollection` or `FlaxT5Layer`. Thus, it can be removed from here and from the `layer_module call`, and the `static_argnums` updated accordingly.